src/services/timerFunction.ts

```diff
@@ -0,0 +1,214 @@
+import moment, { Moment } from "moment";
+import {
+  addFinishedExecution,
+  addOngoingExecution,
+  getOngoingExecutions,
+  setOngoingExecution,
+  removeOngoingExecution,
+} from "./localStorage";
+import { ExecutionStatus, OngoingExecution, FinishedExecution } from "./types";
+
+/**
+ * Coordena o a manipulação de execuções.
+ *
+ * O objetivo é centralizar as funções relacionadas à execuções, de forma a
+ * não poluir as Screens que em que serão utilizadas e que mudanças necessárias
+ * sejam automaticamente propagadas a todas as telas que necessitem trabalhar com
+ * execuções.
+ *
+ */
+
+/** Função responsável por criar uma nova execução */
+export const createExecution = async (cardInfo: any) => {
+  let startTime: Moment = moment();
+  let strStartTime: string = startTime.format("YYYY-MM-DDTHH:mm:ss[Z]ZZ");
+  console.log(
+    "timerFunction.createExecution: Criando execução - Horário atual: ",
+    strStartTime
+  );
+
+  let newExecution: OngoingExecution = {
+    startTime: strStartTime,
+    elapsedTime: 0,
+    latestStartTime: startTime,
+    idPatient: cardInfo.idPatient,
+    role: cardInfo.role,
+    activity: cardInfo.activity,
+    currentState: ExecutionStatus.Uninitialized,
+  };
+  await addOngoingExecution(newExecution);
+  // Chamar add ao storage
+};
+
+/** Função responsável por iniciar contagem de tempo de uma execução
+ * Retorna true se foi possível iniciar, podendo determinar a criação de setInterval()
+ * para fazer update de todas as execuções
+ */
+
+export const initializeExecution = async (id: number) => {
+  // Usar get do storage por indice
```

Comment:
Pelo que entendi recebe o index, certo?

```suggestion
 * para fazer update de todas as execuções.
 * @param index, índice do objeto no array de execuções de atividade
 */

export const initializeExecution = async (index: number) => {
```